### DISCUSSION TITLE
Change Vimtex localleader prefix

### DISCUSSION THREAD

**Discussion by weberam2** [LazyVim/LazyVim#2886] [Q&A]
Hello,

I installed vimtex through the Extras. Now I would like to configure it a bit.

My understanding is to create a vimtex.lua file in my plugins folder

I tried the following, but no luck:

```
return {
  "lervag/vimtex",
  config = function()
    vim.g.vimtex_view_method = "zathura"
    vim.g.vimtex_context_pdf_viewer = "zathura"
    vim.g.vimtex_view_general_viewer = "zathura"
    vim.g.vimtex_mappings_prefix = "<leader>v"
  end,
}
```

**Comment by jyuan0** [comment]
Change `config` to `init`. From the [VimTex README](https://github.com/lervag/vimtex?tab=readme-ov-file#installation):
```lua
-- lazy.nvim
{
  "lervag/vimtex",
  init = function()
    -- Use init for configuration, don't use the more common "config".
  end
}
```

If I'm not mistaken, in general with `lazy.nvim`, you should use an `init` function to set `vim.g` globals.

**Reply by weberam2** [reply]
Thank you!
